sdk/src/resource/resource_detector.cc

```diff
@@ -21,6 +23,10 @@ namespace resource
 
 const char *OTEL_RESOURCE_ATTRIBUTES = "OTEL_RESOURCE_ATTRIBUTES";
 const char *OTEL_SERVICE_NAME        = "OTEL_SERVICE_NAME";
+/**
+ * This is the file path from where we can get container.id
+*/
+const char *C_GROUP_PATH = "/proc/self/cgroup";
```

Comment:
Follow Google style guide for const variable naming - https://google.github.io/styleguide/cppguide.html#Constant_Names